src/components/atoms/Logo/Logo.tsx

```diff
@@ -0,0 +1,5 @@
+import { MainLogoSvg } from '@camp/design';
+
+export const Logo = () => {
+  return <MainLogoSvg />;
+};
```

Comment:
[we decided not to do this for now ](https://github.com/fullstacksjs/the-charity-frontend/pull/27#discussion_r981143516)

Reply:
I can fix this after we merge it